src/server_manager/package.json

```diff
@@ -17,6 +17,7 @@
     }
   },
   "dependencies": {
+    "@sentry/browser": "^5.15.5",
```

Comment:
`@sentry/electron` bundles `@sentry/browser`, so we can use it as a transitive dependency to avoid version collisions. I believe that's what electron docs recommend.

Reply:
I need to depend on the browser version in the code, so I should have it explicit here. I can't use the Electron version without adding libraries to pretend it's Node. We can hard-code a version is needed, but it seems to be working.